csrc/ir/internal_nodes.h

```diff
@@ -129,6 +129,59 @@ class IndexSelectOp : public Expr {
   }
 };
 
+class IndexPutAccumulateOp : public Expr {
+ public:
+  using Expr::Expr;
+
+  // logical ID groups of IndexPutAccumulateOp
+  // args:
+  //     acc   [ ID_indexed_g0, ID_g0 ]
```

Comment:
Indexed IDs in the existing indirect access ops mean those IDs that are indirectly accessed IDs. Here, `ID_indexed_g0` is the indexed ID, but `ID_indexed_g1` doesn't seem so, does it?